mars/core/operand/core.py

```diff
@@ -490,7 +488,18 @@ def execute(results: Dict[str, Any], op: OperandType):
             try:
                 result = executor(results, op)
                 succeeded = True
-                op_executed_number.record(1, {"op": op.__class__.__name__})
+                global op_executed_number
```

Comment:
This seems not easy to use compared to previsous API

Reply:
Can we update the metrics Counter/Guage instance when metris are init?

Reply:
> Can we update the metrics Counter/Guage instance when metris are init?

Yes, good idea. I'll fix it.